extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/MqttThingHandlerFactory.java

```diff
@@ -42,10 +43,11 @@
  *
  * @author David Graeff - Initial contribution
  */
-@Component(service = ThingHandlerFactory.class)
+@Component(immediate = true, service = ThingHandlerFactory.class)
```

Comment:
why is this `immediate` now? should be lazy as all other ThingHandlerFactories I guess.